task1.js

```diff
@@ -0,0 +1,69 @@
+const choices=['Rock','Paper','Scissor'];
+var user_scores=0,computer_score=0;
+function rock()
+{
+    let computer_choice=Math.floor(Math.random() * choices.length);
+    console.log(choices[computer_choice])
+
+    if(choices[computer_choice]=='Rock')
+    {
+        // console.log("helllllllo")
```

Comment:
discard the unwanted/ commeted out code while you push the code. make it a habit of checking the code you push.

Reply:
Okay sir